src/20230817_AaveV3_Eth_SDAIOnboarding/AaveV3_Ethereum_SDAIOnboarding_20230817.sol

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3Ethereum, AaveV3EthereumAssets, IPoolConfigurator} from 'lib/aave-address-book/src/AaveV3Ethereum.sol';
+import {AaveV3PayloadEthereum, IEngine, Rates, EngineFlags} from 'lib/aave-helpers/src/v3-config-engine/AaveV3PayloadEthereum.sol';
+import {IV3RateStrategyFactory} from 'lib/aave-helpers/src/v3-config-engine/IV3RateStrategyFactory.sol';
+
+/**
+ * @title sDAI onboarding
+ * @author Marc Zeller (@marczeller - Aave Chan Initiative)
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0xb41600e3f74bfa520549106fee4d21e7f674c5a79fbf1f7ca16947563474a4d7
+ * - Discussion: https://governance.aave.com/t/arfc-sdai-aave-v3-onboarding/14410
+ */
+contract AaveV3_Ethereum_SDAIOnboarding_20230817 is AaveV3PayloadEthereum {
+  address public constant SDAI = 0x83F20F44975D03b1b09e64809B757c47f942BEeA;
+  address public constant SDAI_PRICE_FEED = 0xb9E6DBFa4De19CCed908BcbFe1d015190678AB5f;
```

Comment:
Looks like this oracle, uses the dai chainlink feed and then multiplies it with the savings rate of sDai.
Can you double check its correct @rustboyar (as you have more exp with this)